packages/persist/lib/records.ts

```diff
@@ -125,20 +125,24 @@ export async function persistRecords({
             updatedResults
         });
 
-        const recordsSizeInBytes = recordsData.reduce((acc, record) => {
+        const recordsSizeInBytes = Buffer.byteLength(JSON.stringify(records), 'utf8');
```

Comment:
sugg: can we use req.contentLength or something like this to avoid the extensive stringify? if not, no worries

Reply:
I quickly check and `stringify` represents very much nothing compared to the round trip between runner<->persist or saving to the db.
I am gonna keep it like this is for now. We can revisit if it ever becomes a bottleneck